src/pages/o/[orgId]/campaigns/[campId].tsx

```diff
@@ -0,0 +1,44 @@
+import { 
+    GetServerSideProps,
+    NextPageContext,
+} from 'next';
+
+export const getServerSideProps : GetServerSideProps = async (context : NextPageContext) => {
+    const { orgId } = context.params;
+    const { campId } = context.params;
```

Comment:
It would be prettier to write this on one line, i.e. `const { campId, orgId } = context.params;`.